test/src/ui/js/BMTestUtils.js

```diff
@@ -65,6 +65,8 @@ BMTestUtils.getAllElements = function() {
 BMTestUtils.deleteEnvMessage = function() {
   delete Env.message;
   $('#env_message').remove();
+  $('#c_body').remove();
+  $('#container').remove();
```

Comment:
Hmm, i'm not sure this is right --- this function used to remove the div containing `#env_message`.  Now it removes the div containing the whole page contents.  Let me try this out and understand what tests fail when you don't add this, because at a glance it seems like it's circumventing the point of the tests to see if various functions are creating things in wrong parts of the page.  (OTOH, maybe those tests are just OBE.  Like i said, i'll take a look.)